test/source_dialogs_test.rb

```diff
@@ -7,7 +7,33 @@
 describe Yast::SourceDialogs do
   subject { Yast::SourceDialogs }
 
+  ZYPP_URL_MAPPINGS = [
```

Comment:
true, originally it was mapping and then I realize that pre and post process is iso scheme only